### PR TITLE
bltool 0.2.1 (new formula)

### PR DESCRIPTION
**Pull request by jawshooah**

### CODE REVIEW

File: Library/Formula/bltool.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  url "https://github.com/ToxicFrog/bltool/releases/download/v0.2.1/bltool-0.2.1.zip"
+  sha256 "dceb5812058699b726bcd43a87c703eba39527959f712ef262919e0e14d6608e"
+
+  def install
+    libexec.install "bltool.jar"
+    bin.write_jar_script libexec/"bltool.jar", "bltool"
+  end
+
+  test do
+    assert shell_output("#{bin}/bltool --help formats").include? "backloggery"
```
Comment (DomT4) [inline Library/Formula/bltool.rb]: I'd rather make this `assert_match /backloggery/, shell_output("#{bin}/bltool --help formats")`. Does the same thing - Just more explicitly and it's the style we're dominantly using recently.

Comment (jawshooah) [inline Library/Formula/bltool.rb]: Cool, I'll make the change.

**Comment by jawshooah** [comment]
@DomT4 Test has been updated as you suggested.

**Comment by DomT4** [comment]
Thanks! The upstream project looks tentatively okay to me (We have a few [requirements](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Acceptable-Formulae.md#niche-or-self-submitted-stuff)), but I'll leave one of @Homebrew/owners to review.

File: Library/Formula/bltool.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    else
+      bltool_jar = "bltool.jar"
+    end
+
+    libexec.install bltool_jar
+    bin.write_jar_script libexec/File.basename(bltool_jar), "bltool"
+  end
+
+  test do
+    assert_match /backloggery/, shell_output("#{bin}/bltool --help formats")
```
Comment (mikemcquaid) [inline Library/Formula/bltool.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Comment by jawshooah** [comment]
@mikemcquaid Done.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**